MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,221 @@
+        
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        let width = UIScreen.main.bounds.width
+        flowLayout.itemSize = CGSize(width: (width / 3) - 32, height: 50)
+        collectionView = UICollectionView(
+            frame: CGRect(
+                x: 0,
+                y: 0,
+                width: view.bounds.width,
+                height: view.bounds.height),
+            collectionViewLayout: flowLayout)
+        
+        view.addSubview(collectionView)
+        
+        collectionView.register(MyCollectionViewCell.self, forCellWithReuseIdentifier: MyCollectionViewCell.cellIdentifier)
+        collectionView.dataSource = self
+        collectionView.autoPinEdge(toSuperviewEdge: .leading, withInset: 16)
+        collectionView.autoPinEdge(toSuperviewEdge: .trailing, withInset: 16)
+        collectionView.autoPinEdge(.top, to: .bottom, of: descriptionLabel, withOffset: 27)
+        collectionView.autoPinEdge(toSuperviewEdge: .bottom)
+    }
+    
+    private func styleViews() {
+        view.backgroundColor = .white
+        
+        ratingLabel.font = UIFont.boldSystemFont(ofSize: 16)
+        ratingLabel.textColor = .white
+
+        userScoreLabel.textColor = .white
+        userScoreLabel.font = UIFont.systemFont(ofSize: 14)
+
+        titleLabel.textColor = .white
+        titleLabel.font = UIFont.boldSystemFont(ofSize: 24)
+
+        releaseYearLabel.textColor = .white
+        releaseYearLabel.font = UIFont.systemFont(ofSize: 24)
+
+        categoriesLabel.textColor = .white
+
+        durationLabel.font = UIFont.boldSystemFont(ofSize: 14)
+        durationLabel.textColor = .white
+        
+        dateLabel.textColor = .white
+
+        overViewLabel.font = UIFont.boldSystemFont(ofSize: 20)
+        overViewLabel.textColor = .black
+    }
+    
+    private func convertDateFormat(sourceDateString : String, sourceDateFormat : String, destinationFormat : String) -> String{
+
+        let dateFormatter = DateFormatter();
+        dateFormatter.dateFormat = sourceDateFormat;
+
+        if let date = dateFormatter.date(from: sourceDateString){
+            dateFormatter.dateFormat = destinationFormat;
+            return dateFormatter.string(from: date)
+        } else {
+            return ""
+        }
+    }
+}
+
+extension MovieDetailsViewController {
+    
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return 1
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return detailsLabel.crewMembers.count
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(
+            withReuseIdentifier: MyCollectionViewCell.cellIdentifier,
+            for: indexPath) as! MyCollectionViewCell
```

Comment:
You should avoid using force unwrapping. Use optional binding instead `guard let`.